src/test/java/uk/co/ramp/event/InfectionEventProcessorTest.java

```diff
@@ -43,7 +45,9 @@
   private InfectionEventProcessor eventProcessor;
   private DiseaseProperties diseaseProperties;
 
+  @Autowired private StandardProperties properties;
   @Autowired private DistributionSampler distributionSampler;
+  @Autowired private StatisticsRecorder statisticsRecorder;
```

Comment:
Remove Autowired annotation and initialise this as: mock(StatisticsRecorder.class)

Reply:
Will do this as a tech debt ticket with some of the other things we discussed yesterday like the gaussian